pyDeltaRCM/init_tools.py

```diff
@@ -33,31 +36,52 @@ def init_logger(self):
             # create the logging file handler
             st = timestr = time.strftime("%Y%m%d-%H%M%S")
             fh = logging.FileHandler("pyDeltaRCM_" + st + ".log")
-
             formatter = logging.Formatter(
                 '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
             fh.setFormatter(formatter)
 
             # add handler to logger object
             self.logger.addHandler(fh)
 
-    def set_defaults(self):
+    def import_files(self):
```

Comment:
cool, so this is the workhouse of the importing. Looks good to me.

My only concern is regarding getting some tests on the importing routine. I.e., testing that it works as expected.

Reply:
For sure. I'll add to develop maybe?

Reply:
Not sure what you mean, sorry. You can add the tests to your PR, and they will get tested on Travis.